cves/CVE-2004-0811.yml

```
@@ -31,7 +31,9 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+  By "merging of the Satisfy directive" one can assume authentication roles with
+  roles contrarry to specified authentication configuration
```

Can you expand on your description a bit more? Right now this is the same as the NVD description and they are looking for more. Also, spelling
```suggestion
  roles contrary to specified authentication configuration
```